### ISSUE TITLE
Laser extending through walls in mp_coop_laser_2

### ISSUE THREAD

**Issue by Luke18033** [StrataSource/Portal-2-Community-Edition#1250]
### Describe the bug

In `mp_coop_laser_2`, if either player places both of their portals on the protruding wall closer to the laser, the beam will go through the portal, activate the first relay, then proceed to extend through the wall and activate the second relay even without any portals there:
![image](https://user-images.githubusercontent.com/19228201/231055585-09193f11-6518-426d-82a4-33bf5fe83586.png)

It doesn't seem to just be clipping through the wall either, because if you noclip inside the wall you can see the beam ends where it hits the wall and then starts again on the other side.
![chaos_iygaZgh9qz](https://user-images.githubusercontent.com/19228201/231055822-a90f8f1c-373e-46f1-b59a-846246eeded1.png)

If you offset the portal, it's even possible for the beam to emerge in a completely different position than it entered!
![image](https://user-images.githubusercontent.com/19228201/231055917-256ab38b-fc61-43ae-827a-36406f2cdce7.png)

It's almost like the game thinks there's a portal there, even though there isn't (and hasn't been since loading the map). Additionally, once when I was aiming the laser around I heard one of the turrets in the next room explode, implying the laser managed to extend even further through the chamber walls, but I wasn't able to actually see this happening or reproduce it.

### Issue Map

mp_coop_laser_2

### To Reproduce

1. Open map
2. Redirect the laser into the portalable wall
3. Place your portals on both sides of it
4. Laser extends through the second wall into the second relay

### Operating System

Tested on Windows 10

**Comment by FrozenDragon0** [comment]
How the heck does a bug like this even occur? It's hilarious but at the same time the heck is even going on here?

**Comment by d0ctorzer0** [comment]
This is still happening as of March 23rd 2026 on staging.
[Video proof](https://youtu.be/xop96Fkg5Zs)

**Comment by lenship2** [comment]
apparently this also happens in singleplayer, as per @cheeseinspace

<img width="1600" height="900" alt="Image" src="https://github.com/user-attachments/assets/e0c4cbff-7e08-415f-971a-8a90f70b9afd" />

**Comment by vrad-exe** [comment]
Yeah, I didn't think it was ever a coop specific issue, mp_coop_laser_2 is just a map that happens to have the right setup to trigger the bug

**Comment by beepster4096** [comment]
This bug is actually in base Portal 2, but it only triggers on surfaces 56u or thinner
